pkgs/development/python-modules/openstep-parser/default.nix

```
@@ -0,0 +1,37 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  pythonOlder,
+  pytestCheckHook,
+  setuptools,
+}:
+
+buildPythonPackage rec {
+  pname = "openstep_parser";
+  version = "1.5.4";
+  pyproject = true;
+
+  disabled = pythonOlder "3.7";
+
+  src = fetchFromGitHub {
+    owner = "kronenthaler";
+    repo = "openstep-parser";
+    rev = "a62840b001a75a59cc2c202831e93f6a014f06ff";
```

The source is [tagged](https://github.com/kronenthaler/openstep-parser/tags).

```suggestion
    rev = "refs/tags/${version}";
```